src/cljdoc/server/pedestal.clj

```diff
@@ -486,7 +486,7 @@
                ctx))}))
 
 (def build-static-resource-map
-  "Extracts all static resource names (content-hashed by Parcel) from the cljdoc.html file.
+  "Extracts all static resource names (content-hashed by esbuild) from the cljdoc.html file.
```

Comment:
Now that we're not using parcel the html file doesn't get updated with the correct filenames, so we need to change how we make this static resource map to not rely on the html file.